src/getAppSyncConfig.js

```diff
@@ -68,6 +69,7 @@ export default function getAppSyncConfig(context, appSyncConfig) {
             region,
             accessKeyId,
             secretAccessKey,
+            sessionToken,
```

Comment:
@Scale93 Let's do one thing: pass down the full `context.options.dynamoDb` object here, just leaving the `tableName` override. That way we will avoid similar issues in the future with other options.

Reply:
Ok It's a good advice!
Do you think that an update to README / Documentation could be useful?

![image](https://user-images.githubusercontent.com/36473880/110144296-76888b00-7dd8-11eb-9bc9-65138f1d9076.png)

Reply:
@Scale93 Absolutely! Good catch. maybe leave those as examples as they are also very common use-cases. But add a note that any configuration property from [the DynamoDB SDK](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/DynamoDB.html#constructor-property) is accepted.

Reply:
@bboure I have made the change suggested by you passing down the full `context.options.dynamoDb` and I have updated README doc!